virt/virt-2-4-release-notes.adoc

```diff
@@ -122,6 +122,8 @@ single sidebar menu item labeled *Virtualization*. When you click *Virtualizatio
 
 [id="virt-2-4-known-issues"]
 == Known issues
+* If you enable a MAC address pool for a namespace by applying the KubeMacPool label, do not use the `io` attribute for virtual machines in a namespace. (link:https://https://bugzilla.redhat.com/show_bug.cgi?id=1869527[*BZ#1869527])
```

Comment:
Maybe it should be in "the" namespace, since the issue would only shown in those namespace that are opted-in? Other than that, it looks good.

Reply:
Made the content change. Moving to peer review.